GameStore.API/Controllers/CommentsController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using GameStore.Application.Interfaces;
+using GameStore.Application.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GameStore.API.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class CommentsController : ControllerBase
+{
+    private readonly ICommentService _commentService;
+
+    public CommentsController(ICommentService commentService)
+    {
+        _commentService = commentService;
+    }
+
+    [HttpPost("/api/game/{gamekey}/newComment")]
```

Comment:
no need to specify `/api` for actions when you have it in controller route. Try to remove it

Reply:
If i try to remove /api prefix  in my requests they all return 404 not found.

Reply:
Try `game/{gamekey}/newComment`

Reply:
Yeah i know, but then it will be api/comments/game/...... but in the task it's api/game/...